client/js/lounge.js

```diff
@@ -962,16 +969,19 @@ $(function() {
 		}
 
 		for (var i in nicks) {
-			words.push(nicks[i]);
+			if (firstWord){
+				words.push(nicks[i]+": ");
+			} else {
+				words.push(nicks[i]+" ");
+			}
 		}
-
 		sidebar.find(".chan")
-			.each(function() {
-				var self = $(this);
-				if (!self.hasClass("lobby")) {
-					words.push(self.data("title"));
-				}
-			});
+		.each(function() {
```

Comment:
Please remove this change, indentation is not related to the subject of your PR.